sway/tree/container.c

```diff
@@ -99,7 +99,7 @@ struct sway_container *container_create(struct sway_view *view) {
 	//   - border
 	//     - border top/bottom/left/right
 	//     - content_tree (we put the content node here so when we disable the
-	//       border everything gets disabled. We only render the content iff there
+	//       border everything gets disabled. We only render the content if there
```

Comment:
I think this means "if and only if": https://en.wikipedia.org/wiki/If_and_only_if